test/browser/msw-api/setup-worker/scenarios/iframe-isolated-response/one.html

```diff
@@ -0,0 +1,34 @@
+<script>
+  const { worker, http } = window.parent.msw
+
+  function isolatedResolver(resolver) {
```

Comment:
@yannbf, hi 👋 Let's try using this utility in the Storybook add-on and see if it provides us with the context isolation when rendering multiple stories on the same page (in iframes). I hope it does! 

The important part here is so this utility runs in the iframe's context, not in the parent context.